go/Makefile

```diff
@@ -11,6 +11,11 @@ TARGET_DIR := $(GLIDE_FFI_PATH)/target
 UNAME := $(shell uname)
 ARCH := $(shell uname -m)
 
+# Set default values for GLIDE_NAME and GLIDE_VERSION
+# GLIDE_VERSION is modifiable by changing the env var, to allow for changing it during CD
```

Comment:
add something like "GLIDE_VERSION is automatically set during the deployment workflow based on the value defined in go-cd.yml. For local builds, you can manually specify the version using `GLIDE_VERSION=<version> make build`"

(make sure that it works)